tests/framework/installer/ceph_installer.go

```diff
@@ -258,6 +258,21 @@ func (h *CephInstaller) CreateCephCluster() error {
 		return errors.Errorf("failed to create custom ceph.conf. %+v", err)
 	}
 
+	customCSISettings := map[string]string{
+		"ceph.conf": "[global]\nauth_client_required = cephx",
```

Comment:
Does ceph parse the `\n`? How about this?
```suggestion
		"ceph.conf": `[global]
auth_client_required = cephx`,
```

Reply:
configmap will get created as required. I have tested this manually and even E2E is also working as this config is mounted to csi pods and it's also used for PVC creation etc.